### ISSUE TITLE
Fix broken image URLs when quarto markdown was in subfolder of content dir

### ISSUE THREAD

**Issue by lmuenter** [lmuenter/pelican-quarto#3]
Currently, Image URLs incorrectly account for nested structures created during the built process.
For instance, if your .qmd file resides in `content/reports/myreport.qmd`, quarto will produce a `output/reports/myreport-files/figure-html/` folder, but reference `output/myreport-files/figure-html/` as the parent folder of the figure.

**Comment by lmuenter** [comment]
The problem appears to be related to how Quarto handles relative paths for files written to stdout. When using the command:

```sh
quarto render --output-file - <file>
```

Quarto seems to assume that the generated HTML will be placed at the root of the report files, like so:

```
output/
    reports/
        myreport-data/
        report.html
```

However, in the Pelican-Quarto setup, the actual structure doffers:

**Content folder:**
```
content/
    reports/
        myreport/
            data/
            data-raw/
        myreport.qmd
```

**Output:**
```
output/
    reports/
        myreport-data/
pelican-slug-for-report.html
```

This leads to two main issues:

1. Internal references to image files are broken.
2. The Pelican slug needs to be used consistently in the output folder.

To address these problems, iwe cpuld xircumenvent writing to stdout. Instead, the `Quarto.run` method could create a temporary Quarto HTML file at the root of `output/`, parse it, remove it, and then return the result. This approach would resolve both issues, as Quarto may rename the report data folder the Pelican slug.


Edit: . [Here's](https://github.com/quarto-dev/quarto-cli/discussions/2171) an older issue about it. Maybe moving of files is the best approach

**Comment by lmuenter** [comment]
Fixed in v0.3.0.
